src/proxy/NearCachedMapProxy.ts

```diff
@@ -109,12 +109,12 @@ export class NearCachedMapProxy<K, V> extends MapProxy<K, V> {
     protected deleteInternal(keyData: Data): Promise<void> {
         this.nearCache.invalidate(keyData);
         return super.deleteInternal(keyData)
-            .then<void>(this.invalidateCacheEntryAndReturn.bind(this, keyData));
+            .then<void>(() => this.invalidateCacheEntryAndReturn(keyData));
```

Comment:
These binds were useless, and they make type checker job harder

Reply:
Also note that the second param is not passed here. So the `invalidateCacheEntryAndReturn` function signature changed as well